### DISCUSSION TITLE
Is the library compatible with Arduino GFX?

### DISCUSSION THREAD

**Discussion by JKeyRandom** [ImpulseAdventure/GUIslice#503] [Q&A]
Hello everyone!, I was wondering if there is support for the Arduino GFX library, if not, could it be adapted in the future?

There are quite striking screens with ILI6485A chip and GT911 capacitive touch, these already have an esp32 incorporated which makes everything much easier.

[Screen](https://es.aliexpress.com/item/1005004974870929.html?spm=a2g0o.store_pc_allProduct.8148356.19.2f067a42DMorUy&pdp_npi=2%40dis%21EUR%21%E2%82%AC%2040%2C51%21%E2%82%AC%2024%2C31%21%21%21%21%21%40211b88ef16783005300987670efa5f%2112000031242398706%21sh)

[Arduino GFX Library](https://github.com/moononournation/Arduino_GFX)

**Comment by Pconti31** [comment]
@JosekeyRandom Not currently.  Doesn't look hard to support.  Simply creating a GUIslice_drv_arduinogfx.(h cpp) copying a lot from
GUIslice_drv_adagfx.(h cpp)
You should make a request for enhancement in the issues section or if you are technical make your own update and issue a pull request.
 
if @ImpulseAdventure Calvin modifies GUIslice API to use it I might do the same with the Builder.  The only issue with the builder is support for the U8g2 fonts which might be hard depending upon its documentation of its binary bits...

Paul--

**Reply by JKeyRandom** [reply]
@Pconti31 Ok I understand, I'll try to add it to see if it works but I'm afraid that if the builder isn't updated it won't work... On the other hand the GT911 touch driver will also be a problem I imagine. Very grateful for your answer.
